schemas/1.0/maat_application.json

```diff
@@ -10,6 +10,7 @@
     "reference": { "type": "integer" },
     "application_type": { "type": "string", "enum": ["initial"] },
     "submitted_at": { "type": "string", "format": "date-time" },
+    "signed_at": { "type": "string", "format": "date-time" },
```

Comment:
Don't have a strong opinion on this but maybe `declaration_signed_at` is more explicit?
Mostly because `signed_at` might be confused with some kind of cryptographic signing?

Reply:
Making things clearer is always good with me!